src/components/menu-bar/save-status.jsx

```diff
@@ -40,6 +40,14 @@ const SaveStatus = ({
         </div>
     ));
 
+// Listen for Control/Command + S.
+document.addEventListener("keydown", function(e) {
```

Comment:
This cannot go here, because `onClickSave` is only defined inside the render function above. This listener can be attached in the menu-bar.jsx file, add a componentDidMount to add the listener and a componentWillUnmount where you remove the listener. 

[This guide](https://reactjs.org/docs/react-component.html#componentdidmount) might be helpful when dealing with React components